### ISSUE TITLE
Google App Engine

### ISSUE THREAD

**Issue by enriquejosepadilla** [jschneier/django-storages#668]
The documentation @ https://django-storages.readthedocs.io/en/latest/backends/gcloud.html only mentions GKE and GCE. Does this work for GAE, specifically for the newest one that supports python 3?

We are looking to streamline File uploads for projects with our tool @ https://github.com/GoogleCloudPlatform/django-cloud-deploy and came across this.

**Comment by sww314** [comment]
@enriquejosepadilla it should support GAE. The API to Google Cloud Storage is the same. The only issue maybe service accounts - the default GAE account may not have the right permissions.

**Comment by bgrace** [comment]
Yes, it works fine in the 2nd generation standard environment, Python 3 runtime.

**Comment by ppiet** [comment]
👍 yes, I found it to work mostly perfecly, although I couldn't make it work with GCS buckets with uniform access policy. My notes are @ https://pnote.eu/notes/django-app-engine-user-uploaded-files/ , I also filed #909 for the issue with buckets with uniform access policy.

**Comment by pataquets** [comment]
So, @enriquejosepadilla, can the issue be closed?
